candle-nn/src/ops.rs

```diff
@@ -926,3 +926,19 @@ pub fn replication_pad2d(xs: &Tensor, pad: usize) -> Result<Tensor> {
         n => candle::bail!("replication-pad with a size of {n} is not supported"),
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct Identity {}
```

Comment:
If we don't plan on providing any argument, `pub struct Identity;` seems very slightly better.